runtime/rasdump/javadump.cpp

```diff
@@ -1052,14 +1052,31 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	/* Write the running mode data */
 	_OutputStream.writeCharacters("1CIRUNNINGAS   Running as ");
 
+	/* If JITServer enabled, print running mode as JITServer and client UID*/
+#ifdef JITSERVER_SUPPORT
+	if (jitConfig->isClient == 1) {
+		_OutputStream.writeCharacters("a JITServer Client");
+	} else {
+		_OutputStream.writeCharacters("a JITServer Server");
+	}
 	/* NB : I can't find any code for making this decision in the existing implementation */
-#ifdef J9VM_BUILD_J2SE
+#elif J9VM_BUILD_J2SE
 	_OutputStream.writeCharacters("a standalone");
 #else
 	_OutputStream.writeCharacters("an embedded");
 #endif
 	_OutputStream.writeCharacters(" JVM\n");
 
+#ifdef JITSERVER_SUPPORT
+	if (jitConfig->isClient == 1) {
+		char valueString[_MaximumGPValueLength]; 
+		sprintf(valueString, "%llu", (unsigned long long) jitConfig->clientUID);
```

Comment:
I looked at some other examples and this is how we write a 64-bit unsigned value:
`_OutputStream.writeInteger64(j9time_nano_time(), "%llu");`